src/libraries/System.Globalization/tests/Invariant/InvariantMode.cs

```diff
@@ -627,6 +627,18 @@ public void TestSortVersion(string cultureName)
             Assert.Equal(version, new CultureInfo(cultureName).CompareInfo.Version);
         }
 
+        [Fact]
+        public void TestSortKey_ZeroWeightCodePoints()
+        {
+            // In the invariant globalization mode, there's no such thing as a zero-weight code point,
+            // so the U+200C ZERO WIDTH NON-JOINER code point contributes to the final sort key value.
+
+            CompareInfo compareInfo = CultureInfo.InvariantCulture.CompareInfo;
+            SortKey sortKeyForEmptyString = compareInfo.GetSortKey("");
+            SortKey sortKeyForZeroWidthJoiner = compareInfo.GetSortKey("\u200c");
+            Assert.NotEqual(0, SortKey.Compare(sortKeyForEmptyString, sortKeyForZeroWidthJoiner));
+        }
```

Comment:
What is the expected result here?
```csharp
        Console.WriteLine(SortKey.Compare(
            CultureInfo.InvariantCulture.CompareInfo.GetSortKey("\uFFFD"),
            CultureInfo.InvariantCulture.CompareInfo.GetSortKey("\u200C")));
```

Reply:
I believe that on NLS (Windows) they're equal and that on ICU (non-Windows) they're unequal, but I'm not current on a box where I can easily check.

Reply:
I checked this today - my statement above is correct. Under the default / invariant culture, on ICU `"\ufffd"` and `"\u200c"` are not equal, but using Windows's `CompareStringEx` they are equal.